definitions/infra-ibmmq_manager/summary_metrics.yml

```diff
@@ -2,30 +2,38 @@ connections_count:
   title: Connections
   unit: COUNT
   query:
-    select: latest(ibmmq_qmgr_connection_count)
+    select: average(ibmmq_qmgr_connection_count) AS 'Connections'
     from: Metric
     eventId: entity.guid
 
-queue_uncommitted_messages:
-  title: Uncommitted messages average.
+queue_count:
+  title: Queues
   unit: COUNT
   query:
-    select: average(ibmmq_queue_uncommitted_messages)
+    select: uniqueCount(queue) AS 'Queues'
```

Comment:
we don't support aliases in the summary metrics

Reply:
👍 Removed. Thanks!